forge-gui/res/cardsfolder/upcoming/reverse_the_polarity.txt

```diff
@@ -0,0 +1,9 @@
+Name:Reverse the Polarity
+ManaCost:1 U U
+Types:Instant
+A:SP$ Charm | Cost$ 1 U U | Choices$ DBCounter,DBSwitch,DBUnblockable

```

Comment:
No need for Cost param unless it is different than ManaCost

Reply:
It's weird then, because I've seen a lot of scripts that are written exactly like this.

Reply:
that's exactly it
just nobody bothered enough to make that big cleanup yet